README.md

```diff
@@ -1433,6 +1433,7 @@ Table of Contents
    * [counter.dev](https://counter.dev) — Web analytics made simple and therefore privacy friendly. Free or pay what you want by donation.
    * [PostHog](https://posthog.com) - Full Product Analytics suite free for up to 1m tracked events per month
    * [Uptrace](https://uptrace.dev) - Distributed Tracing Tool that helps developers pinpoint failures and find performance bottlenecks. Has a free plan, offers a free Personal subscription for open source projects, and has an open source version.
+   * [InterServer](https://www.interserver.net/webhosting/student-webhosting.html/) [FREE] - Standard Web Hosting Package for Free for the first year for students.
```

Comment:
This one is a bit too limited being 1st year students only for this list. I think we need to aim at broader audiences for this list.

Reply:
actually it's not for 1st year students only.. its free for only 12 month.(for the first yr..not for the first yr students)

Reply:
OK, still, too limited I think

Reply:
let me open another pull..i accidentally added "[FREE]" text.. which was for another repo.

Reply:
> OK, still, too limited I think

how this can still limited? :/

Reply:
I myself own & run a hosting company offering FreeHosting for a lifetime, it isn't that costly to run one. 

Ripienaar thinks it as(assumed) limited since most webhostings offer free services for an lifetime/unlimited period.

Reply:
might be.

Reply:
Its limitd not in features but because its for 1 year and only for students and checks an .edu domain.  The demographic of people for who this is useful is too limited and out of scope of the aim of the list.